app/views/casa_cases/_form.html.erb

```diff
@@ -108,12 +108,8 @@
         </div>
       <% end %>
       <% if Pundit.policy(current_user, casa_case).update_contact_types? %>
-        <%= render "case_contacts/form/contact_types",
-                   form: form,
-                   model: casa_case,
-                   contact_type_groups: @contact_types.map(&:contact_type_group).sort_by(&:name).uniq,
-                   selected_case_contact_types: [],
-                   checkbox_tag_name: "casa_case[casa_case_contact_types_attributes][][contact_type_id]" %>
+        <!-- TODO - need to refactor this -->
+        <%= render "case_contacts/form/contact_types", form: form, options: @contact_type_options, selected_items: @contact_type_selected_items %>
```

Comment:
This is just a general comment. But like what even is the functionality on this page? 
![image](https://github.com/rubyforgood/casa/assets/14540596/770627ea-4f41-44be-9dfa-3f64195e1aa0)

Selecting stuff there doesn't seem to affect future case contacts? What does `contact_type` selection even do there?